sdk/src/integrationTest/java/IntegrationTestEnv.java

```diff
@@ -215,7 +215,8 @@ public void nextNode(Map<String, AccountId> outMap) throws Exception {
                     nodes.remove(index);
                     outMap.put(node.getKey(), node.getValue());
                     return;
-                } catch (Throwable ignored) {
+                } catch (Throwable err) {
+                    System.out.println(err);
```

Comment:
Can we change this to `System.err.println()`

Reply:
Sure, makes sense.